src/lib/i18n/dictionaries/dictionary.json

```
@@ -5740,5 +5740,9 @@
   "taskBoard.tasksDescription": "Complete tasks to earn Love Charm.",
   "taskBoard.tasksDescriptionTwo": "Love Charm can be used to purchase special items in the rewards shop",
   "taskBoard.otherWays": "Other ways to earn Love Charm",
-  "taskBoard.howToRefer": "How to refer a friend?"
+  "taskBoard.howToRefer": "How to refer a friend?",
+  "marketplace.bought": "Bought",
+  "marketplace.sold": "Sold",
```

```suggestion
  "marketplace.boughtListing": "Bought Listing",
  "marketplace.madeListing": "Made Listing",
```

Do we wanna standardise with the offer wording?